weatherrouting/polar.py

```diff
@@ -171,8 +171,9 @@ def getTWARoutage(self,tws,twa):
 		return twa
 
 		
-	def maxReachDistance(self,p,twd,tws):
+	def maxReachDistance(self, p, twd, tws, speed = None):
```

Comment:
It seems this function can be simplified in 
```
	def maxReachDistance(self, p, speed):
		dt = (1. / 60. * 60.)
		maxp = utils.routagePointDistance (p[0], p[1], speed * dt, 1)
		return utils.pointDistance(p[0], p[1], maxp[0], maxp[1])
```

Reply:
Ok